airbyte-webapp/src/packages/cloud/views/auth/Auth.module.scss

```diff
@@ -26,10 +27,12 @@
   overflow: hidden;
 }
 
-@media (min-width: 992px) and (min-height: 720px) {
-  .container {
-    flex-direction: row;
-    height: 100%;
```

Comment:
:information_source: Moving those out of the media query was large part of the fix to get rid of the squishing behavior.